frontend/pnpm-lock.yaml

```diff
@@ -1,4 +1,4 @@
-lockfileVersion: '6.0'
+lockfileVersion: '6.1'
```

Comment:
a reminder for me to undo this

Reply:
If you do normal `pnpm install` it updates the lockfile, so if the installed version of pnpm uses a newer lockfile version it will update it. The best way to avoid that is doing `pnpm install --frozen-lockfile`.